plugins/logs/plugin_test.go

```diff
@@ -482,6 +482,106 @@ func TestPluginRequeBufferPreserved(t *testing.T) {
 	}
 }
 
+func TestPluginRateLimitInt(t *testing.T) {
+	ctx := context.Background()
+
+	numDecisions := 1 // 1 decision per second
+
+	fixture := newTestFixture(t, func(c *Config) {
+		limit := float64(numDecisions)
+		c.Reporting.MaxDecisionsPerSecond = &limit
+	}, func(c *Config) {
+		limit := int64(300)
+		c.Reporting.UploadSizeLimitBytes = &limit
```

Comment:
Why do we need this here?

Reply:
The default upload size is 32768 bytes and our test input is pretty small. In order to serialize the event via [this](https://github.com/open-policy-agent/opa/blob/master/plugins/logs/encoder.go#L47) logic we do this.

Reply:
Ah, thanks 👍